### DISCUSSION TITLE
Need Help Installing. The only mod I'm trying to get it to work with is Enhanced Edition Redux Next Gen Beta

### DISCUSSION THREAD

**Discussion by ZukoIsBack** [Aelto/W3_RandomEncounters_Tweaks#189] [Q&A] [answered]
My game works fine with Enhanced Edition Redux Next Gen Beta, but it breaks when I try to add this.

This screenshot shows my merges. I followed the in depth install guide on merging, but there is one merge that popped up that wasn't in the guide called game\player\playerInput.ws. I've tried all possible combination of merges, and none work.

![Screenshot 2023-03-13 053754](https://user-images.githubusercontent.com/127753102/224708155-04b4d080-22ca-4721-86d1-ed87d902140d.png)

![Screenshot 2023-03-13 055410](https://user-images.githubusercontent.com/127753102/224709160-233731ed-8a32-4c4d-b920-e6b209ef6e61.png)

When I try to compile, I get this error. I think this is the bigger issue.

Error [mod_sharedutils_custombossbar]local\sharedutils\custombossbar\globals.ws(10): Could not find function 'show'
Error [mod_sharedutils_custombossbar]local\sharedutils\custombossbar\globals.ws(24): Could not find function 'setCurrentPercentage'
Error [mod_sharedutils_custombossbar]local\sharedutils\custombossbar\globals.ws(38): Could not find function 'hide'
Error [mod_sharedutils_storage]local\storage\main.ws(11): 'SU_storage' is not a member of '&handle:CPlayerInput'
Error [mod_sharedutils_storage]local\storage\main.ws(14): 'SU_storage' is not a member of '&handle:CPlayerInput'
Error [mod_sharedutils_storage]local\storage\main.ws(17): 'SU_storage' is not a member of '&handle:CPlayerInput'

Warning [modw3ee]core\math.ws(590): Global native function 'IntToUint32' was not exported from C++ code.
Warning [modw3ee]core\math.ws(591): Global native function 'Uint32ToInt' was not exported from C++ code.
Warning [modsharedimports]engine\environment.ws(30): Global native function 'EnableDebugOverlayFilter' was not exported from C++ code.
Warning [modsharedimports]engine\environment.ws(32): Global native function 'EnableDebugPostProcess' was not exported from C++ code.
Warning [content0]engine\showflags.ws(11): Global native function 'DebugSetEShowFlag' was not exported from C++ code.

Does anyone know what to do?

**Comment by Aelto** [comment, marked as answer]
Hi !

You're correct i forgot to mention the playerInput.ws file. Merge all 3 mods together for this one.

Your errors however aren't all from that merge, as you can see from your script-merger the sharedutils custombossbar mod is written in grey. That's because you set a custom priority to your EE, causing the game to ignore the default alphabetical ordering. Update your priorities to include the sharedutils custom boss bar to load before (a lower priority number than EE) and that would work!

**Reply by ZukoIsBack** [reply]
Thanks you so much! Can't wait to play with it!
